README.md

```diff
@@ -558,6 +560,16 @@ docker_compose { '/tmp/docker-compose.yml':
 
 Give options to the ```docker-compose up``` command, such as ```--remove-orphans```, by using the ```up_args``` option.
 
+To supply multiple overide compose files add the following to the manifest file:
+
+```puppet
+class {'docker::compose':
```

Comment:
shouldn't this be docker_compose ?
class docker::compose is used just to install docker-compose and does not accept the compose_files parameter

Reply:
@florindragos yes you are absolutely correct , ill change this now.